pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Description
+{ This PR adds ... }
+
+{ This PR fixes ... }
+
+{ This PR removes ... }
+
+## Due Diligence Checklist
+- [ ] Tests
+- [ ] Documentation
+- [ ] Consolidated commits
+- [ ] Relevant labels assigned
+- [ ] Changelog(s) updated
```

Comment:
This is the only part of this template that seems like it should vary depending on the repo. Should I leave this out?

Reply:
I wouldn't mind adopting a habit of adding a changelog to all projects (monorepo or not), this is a good enough excuse for that.